src/core/evaluator.js

```diff
@@ -1546,7 +1535,6 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
         task.ensureNotTerminated();
         timeSlotManager.reset();
         var stop, operation = {}, args = [];
-        let pendingBeginText = false;
```

Comment:
*Please note:* This is actually a bug in the original PR, since this line should be placed e.g. just after https://github.com/mozilla/pdf.js/pull/10041/files#diff-0b94c2e77a5259f7a728122fdbf9f46aL1533 instead to ensure that `pendingBeginText` may never be accidentally reset.